### PR TITLE
Update delegation

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: common/src/reserved.rs [hunk 1]
```diff
@@ -32,22 +32,17 @@ impl ReservedState {
...
             } else {
                 validator_set
-                    .entry(member.public_key.clone())
+                    .entry(member.name.clone())
                     .and_modify(|v| *v += member.consensus_voting_power)
                     .or_insert(member.consensus_voting_power);
             }
         }
-        let mut validator_set: Vec<(PublicKey, u64)> = validator_set
+        // TODO handle error
```
Comment (TomTaehoonKim) [inline common/src/reserved.rs:40]: TODO: ?

Comment (TomTaehoonKim) [inline common/src/reserved.rs:40]:
```suggestion
        // TODO: handle error
```
Plus what error handling needed?

**Review by YoonhoKim0527 — COMMENTED** [review]
LGTM

File: common/src/reserved.rs (hunk 1 above)
Comment (junha1) [reply common/src/reserved.rs:40]: We do unwrap here.

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Pull request merged**